### DISCUSSION TITLE
Improve manpage

### DISCUSSION THREAD

**Discussion by thealio** [leftwm/leftwm#1128] [Ideas]
Hi
I notice that the manpage misses some information about how to configure.

For example, a  in the layout section , the layouts are still reported without double quotes. and the layout_definition settings are not reported.

I'd like to see a more updated and complete manpage, so the user can use it as a guide for configure leftwm, even without  log to the github page

**Comment by VuiMuich** [comment]
Closing this in favor of issue #1129
